<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #9c9c9c;
color: #D8DEE9; </string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
    background: transparent; /* Или убрать фон вообще */
    font-weight: bold;
    font-size: 12pt;
}</string>
   </property>
   <property name="text">
    <string>Клиент</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: #309898;
    color: black;
    border: 1px solid #4C566A;
    border-radius: 5px;
    padding: 6px;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: black;
    background: transparent; /* Или убрать фон вообще */
    font-weight: bold;
    font-size: 12pt;
}</string>
      </property>
      <property name="text">
       <string>Логин</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: black;
    background: transparent; /* Или убрать фон вообще */
    font-weight: bold;
    font-size: 12pt;
}</string>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #309898;
    color: black;
    border: 1px solid #4C566A;
    border-radius: 4px;
    padding: 4px;
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #309898;
    color: black;
    border: 1px solid #4C566A;
    border-radius: 4px;
    padding: 4px;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #309898;
    color: black;
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #81A1C1;
}</string>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #309898;
    color: black;
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #81A1C1;
}</string>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
